test/libsolidity/smtCheckerTests/abi/abi_encode_call_simple_1.sol

```diff
@@ -0,0 +1,19 @@
+contract C {
+	function callMeMaybe(uint a, uint b) external {}
+
+	function abiEncodeSimple(uint x, uint y, uint z) public view {
+		require(x == y);
+		function (uint, uint) external f = this.callMeMaybe;
+		bytes memory b1 = abi.encodeCall(f, (x, z));
+		bytes memory b2 = abi.encodeCall(f, (y, z));
+		assert(b1.length == b2.length); // should hold
+
+		bytes memory b3 = abi.encodeCall(this.callMeMaybe, (3, z));
+		assert(b1.length == b3.length); // should fail
```

Comment:
This is also a false positive, right?

Reply:
ah yea, right. It's true that b1 == b3 is not necessarily true, but the length should be the same